lib/app/features/protect_account/authenticator/views/components/copy_key_card.dart

```diff
@@ -67,27 +64,27 @@ class CopyKeyCard extends HookConsumerWidget {
                     : IndicatorType.dark,
               ),
           SizedBox(height: 20.0.s),
-          Button(
-            minimumSize: Size(148.0.s, 48.0.s),
-            leadingIcon: isCopied.value
-                ? Assets.svg.iconBlockCheckGreen.icon()
-                : Assets.svg.iconBlockCopyBlue.icon(),
-            borderColor: isCopied.value
-                ? context.theme.appColors.success
-                : context.theme.appColors.strokeElements,
-            onPressed: () {
-              if (code.value == null) return;
-              copyToClipboard(code.value!);
-              isCopied.value = true;
-              delayed(() => isCopied.value = false, after: 3.seconds);
-            },
-            label: Text(
-              isCopied.value ? context.i18n.wallet_copied : context.i18n.button_copy,
-              style: context.theme.appTextThemes.body.copyWith(
-                color: context.theme.appColors.primaryText,
+          CopyBuilder(
+            builder: (context, onCopy, isCopied) => Button(
+              minimumSize: Size(148.0.s, 48.0.s),
+              leadingIcon: isCopied
+                  ? Assets.svg.iconBlockCheckGreen.icon()
+                  : Assets.svg.iconBlockCopyBlue.icon(),
+              borderColor: isCopied
+                  ? context.theme.appColors.success
+                  : context.theme.appColors.strokeElements,
+              onPressed: () {
+                if (code.value == null) return;
+                onCopy(code.value!);
+              },
+              label: Text(
+                isCopied ? context.i18n.wallet_copied : context.i18n.button_copy,
```

Comment:
The name for the translation is `context.i18n.wallet_copied`, but we're not on the Wallets page. Consider changing the name.